srcs/class/request.cpp

```diff
@@ -2,133 +2,148 @@
 #include <algorithm>
 #include <string.h>
 
-Route find_route( std::vector<Route> routes, std::string url ) {
+Route find_route(std::vector<Route> routes, std::string url)
+{
 
 	Route route;
-	for ( std::vector<Route>::iterator it = routes.begin(); it != routes.end(); ++it )
+	for (std::vector<Route>::iterator it = routes.begin(); it != routes.end(); ++it)
 	{
-		std::string test_url = finish_by_only_one( url, '/' );
-		if ( strncmp( test_url.c_str(), it->location.c_str(), it->location.size() - 1 ) == 0 )
+		std::string test_url = finish_by_only_one(url, '/');
+		if (strncmp(test_url.c_str(), it->get_location().c_str(), it->get_location().size() - 1) == 0)
```

Comment:
Pourquoi passer par un `strncmp` dans ce cas là?
test_url et get_location sont tous les 2 des strings!
`test_url == it->get_location()` sera beaucoup plus optimisé, clair et propre ;)

```suggestion
		if (test_url == it->get_location())
```

Reply:
Je suis d'accord!

Reply:
Finalement, la clause "it->location.size() - 1 " est ce qui diffère du == ou .compare() régulier, on segfault dans ces cas.